app/assets/stylesheets/helpers/_header.scss

```diff
@@ -516,6 +516,22 @@ $covid-grey: #262828;
   }
 }
 
+.global-bar-title__icon {
+  display: inline-block;
+  display: -ms-flexbox;
```

Comment:
Vertical centering doesn't work in IE11 downwards. I think this is because IE requires an explicit height to be set for vertical centering to work (which I wouldn't recommend doing). Not sure what to suggest.

<img width="570" alt="Screenshot 2020-04-14 at 14 33 39" src="https://user-images.githubusercontent.com/861310/79230813-15249a00-7e5d-11ea-9fe8-a30992dacd9d.png">

Reply:
We seem to have solved this problem for the same link on https://www.gov.uk/coronavirus. Suggest you copy the markup/styles from there.

Reply:
Looks like the solution was adding an explicit height